### PR TITLE
app-profile-update

### PR DESCRIPTION
**Pull request by resmaa**
image
image
image

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/986#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:56]: Between steps 3 and 4 we agreed to add a Warning box about namespace. Below I offered a couple variations of text for you to consider because you're more familiar with the product. Also, should "namespace" be bold?

_You suggested:_ "When adding layers to the App profile using manifest files, specify a **namespace** name to which the resource must belong."

_Another suggestion with slight change:_ "When adding layers to the App profile using manifest files, you must specify a **namespace** name to which the resource belongs."

_Another suggestion:_ "When adding layers to the App profile using manifest files, you must specify a **namespace** name to indicate which resource the layer belongs to."

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:19]:
```suggestion
App Profiles are templates created with preconfigured services required for sandbox cluster deployment. App Profiles provide a way to drive consistency across sandbox clusters. You can create as many profiles as required. An App Profile can be created to meet specific types of Workload on Palette Virtual Clusters. For example, you may create a Development App Profile with <Tooltip trigger={<u>Helm Charts</u>}>Add a Palette App profile Pack from Helm registry charts. The public Palette Pack registry and a few popular Helm chart repositories are already available out of the box. Add additional pack registries or public/private chart registries to Palette. </Tooltip>
```

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:20]:
```suggestion
, <Tooltip trigger={<u>Manifest</u>}>Layers can be constructed using raw manifests to provision Kubernetes resources that are unavailable in Palette or Charts. Pack Manifests provide a pass-through mechanism in which additional Kubernetes resources can be orchestrated onto a cluster and the rest of the stack. Specific integrations may require the creation of Secrets or CustomResourceDefinition (CRDs). Manifest files can be attached to the layer to achieve this. For example, you could attach a Manifest to different database applications.
```

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:45]:
```suggestion
3. **Add Service**: Select a service to start configuring the App Profile from the available services and configure the service based on your requirements. The following services are provided:
```

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:47]:
```suggestion
  * Container Deployment: Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.
```

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile.md:53]:
```suggestion
 * Manifest: Layers can be constructed using raw manifests to provision Kubernetes resources that are unavailable in Palette or Charts. Pack Manifests provide a pass-through mechanism in which additional Kubernetes resources can be orchestrated onto a cluster and the rest of the stack. Specific integrations may require the creation of Secrets or CustomResourceDefinition (CRDs). Manifest files can be attached to the layer to achieve this. 
```

**Review by ritawatson — CHANGES_REQUESTED** [review]
Added Warning box text suggestions. I also looked at other content and made some suggestions. In our team meeting, our team discussed not using tool tips, so I didn't comment on these because you're working on removing them.

**Comment by karl-cardenas-coding** [comment]
@ritawatson I think you mean info boxes. We should only use warning boxes when we warn the user about a potential pitfall or security-related threat.

**Comment by ritawatson** [comment]
When @resmaa  and I discussed this with the developer, we thought warning box because adding a profile fails if the user doesn't provide a namespace. UI team will enforce adding a namespace in a future release and at that time, we can remove the box. I defer to you and Resma as to which type of box is used.

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/986#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/986#pullrequestreview-1183374598" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/05-app-profile.md:46]: @resmaa - this description looks good for containers. Will we have a seperate page or section which talks about attributes of the Container deployment?

You can talk to @shubhamrajvanshi incase you need more info about each supported attribute.

**Comment by resmaa** [comment]
contents merged to "rename" branch

**Pull request closed without merging**
